### PR TITLE
Release ios-v3.3.6

### PR DESCRIPTION
**Pull request by boundsj**
Opening this for review. I don't intend to actually merge it.

cc @jfirebaugh @1ec5

### CODE REVIEW

**Comment by boundsj** [comment]
> I don't intend to actually merge it.

Scratch that -- forgot about the tag. Will merge later. cc @1ec5

File: platform/ios/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
 ## 3.3.5
+* Back ported changes to fix an OpenGL crash. ([#6972](https://github.com/mapbox/mapbox-gl-native/pull/6972))
```
Comment (1ec5) [inline platform/ios/CHANGELOG.md]: > * Fixed a crash that occurred during low-memory situations when multiple instances of MGLMapView were in the view hierarchy. The speculative fix in v3.3.5 has been reverted. ([#6972](https://github.com/mapbox/mapbox-gl-native/pull/6972))

File: platform/ios/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,10 @@
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
 ## 3.3.5
```
Comment (1ec5) [inline platform/ios/CHANGELOG.md]: 3.3.6

**Comment by boundsj** [comment]
Thanks @1ec5. Fixed.

**Comment by boundsj** [comment]
The release [is published](https://github.com/mapbox/mapbox-gl-native/releases/tag/ios-v3.3.6). Merging in https://github.com/mapbox/mapbox-gl-native/pull/6992

**Pull request closed without merging**